### DISCUSSION TITLE
Crazyflie Analog Input

### DISCUSSION THREAD

**Discussion by leszekambro** [bitcraze/discussions#860] [Q&A]
Hello, I would like to ask if there is a possibility to read on Crazyflie platform analog signal in range +/- 5V? I found only information about adc input availability but without any details (eg. resolution etc.). I will be grateful for any help.

**Comment by tobbeanton** [comment]
Yes but you would have to use a resistor divider to get the signal in the 0-3.0V range. Then you can use the deck API [analogRead](https://github.com/bitcraze/crazyflie-firmware/blob/master/src/deck/api/deck_analog.c#L83C10-L83C20). The [maxsonar.c](https://github.com/bitcraze/crazyflie-firmware/blob/master/src/drivers/src/maxsonar.c) could perhaps be used as an example.
